### PR TITLE
DOCSP-37407 Merge Prerequisite Content to Master

### PR DESCRIPTION
**Pull request by ianf-mongodb**
## DESCRIPTION

Content to support pre-requisit requirements of using Relational Migrator with your on-prem database. This content expands on the automatically generated scripts from Relational Migrator and explains each step as if it were a manual task.

## STAGING

https://preview-mongodbianfmongodb.gatsbyjs.io/docs-relational-migrator/DOCSP-37407/prerequisites/

https://preview-mongodbianfmongodb.gatsbyjs.io/docs-relational-migrator/DOCSP-37407/prerequisites/my-sql/

https://preview-mongodbianfmongodb.gatsbyjs.io/docs-relational-migrator/DOCSP-37407/prerequisites/oracle/

https://preview-mongodbianfmongodb.gatsbyjs.io/docs-relational-migrator/DOCSP-37407/prerequisites/postgres/

https://preview-mongodbianfmongodb.gatsbyjs.io/docs-relational-migrator/DOCSP-37407/prerequisites/sql-server/

https://preview-mongodbianfmongodb.gatsbyjs.io/docs-relational-migrator/DOCSP-37407/prerequisites/sybase/

## JIRA

https://jira.mongodb.org/browse/DOCSP-37407

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=65e87cfd11db7b04270e09c0

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/prerequisites.txt [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+Prerequisites
+=============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+The following pages outline the details and specifics of prepping your
```
Comment (ianf-mongodb) [inline source/prerequisites.txt:13]: I elected not to do the summary table here, we could possibly add as a fast follow once some of the major projects are landed for .local this year.

**Comment by ianf-mongodb** [comment]
@tomhollander just tagging you for a last-minute quality check on all the changes before we publish :).

**Review by tomhollander — APPROVED** [review]
This looks amazing!

**Comment by davidsoulmdb** [comment]
Nitpick on Oracle: If your migrating -> If you're migrating

**Comment by davidsoulmdb** [comment]
I suggest you add a database listing or further content to the top-level https://preview-mongodbianfmongodb.gatsbyjs.io/docs-relational-migrator/DOCSP-37407/prerequisites/ as it's very bare?

**Comment by davidsoulmdb** [comment]
For ordering, I notice the titles turn out quite long with the lede being the last word in the sentence. Perhaps you could address both with a naming scheme from 'Configure Migration Prerequisites for SQL Server' -> 'SQL Server Migration Prerequisites' or similar?

**Comment by ianf-mongodb** [comment]
Hi @davidsoulmdb I have included the first two comments for the nit and filling out the stub page a bit more, you can see the updates in [this commit](https://github.com/mongodb/docs-relational-migrator/pull/99/commits/60d36bf990aa39c21df5ab9bfdec85ceafc60d67). For the page naming, [our style guide dictates](https://www.mongodb.com/docs/meta/style-guide/information-types/prototypes/task/#task-page-prototype) an action/verb driven page title for task pages. I originally had what you suggested for the page titles, but we decided to update the page titles for consistency/style considerations.

**Pull request merged**